website/source/docs/providers/aws/d/availability_zones.html.markdown

```diff
@@ -0,0 +1,39 @@
+---
+layout: "aws"
+page_title: "AWS: aws_availability_zones"
+sidebar_current: "docs-aws-datasource-availability-zones"
+description: |-
+    Provides a list of availability zones which can be used by an AWS account
+---
+
+# aws\_availability_zones
```

Comment:
Does this second underscore also need to be escaped, or will Markdown ignore it because there's an odd number of them? (In other files I've seen all the underscores escaped and have been following that pattern, but not sure if that's actually necessary...)